CONTRIBUTING.md

```diff
@@ -21,3 +21,52 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
+
+## StableHLO Opset Changes
+
+All changes to the StableHLO opset including new ops, types, or attributes must
+be reviewed via an RFC. We aim for StableHLO opset changes to take ~2 weeks
+if feedback is actively addressed. This allows adequate time for the community
+to review all proposals.
+
+### 1. Write an RFC
+
+An RFC should outline the proposed spec changes, as well as the rationale, and
+alternatives considered if relevant. This can be shared as a markdown file in
+the [`rfcs/`](https://github.com/openxla/stablehlo/tree/main/rfcs) directory and
+shared as a PR.
+
+For example, see the [`collective_broadcast` RFC](https://github.com/openxla/stablehlo/pull/1809).
+
+### 2. Notify OpenXLA Discuss
+
+To signal boost your RFC, post on [OpenXLA Discuss](https://groups.google.com/a/openxla.org/g/openxla-discuss).
+This will ensure the proper reviewers see the RFC. While there is no formal
+process for posts, we tend recommend keeping RFC feedback on the PR to keep
+feedback centralized in the repository.
+
+For example, see the [`collective_broadcast` post](https://groups.google.com/a/openxla.org/g/openxla-discuss/c/Q7JFyoiVFPU/m/dUH_LmJlCgAJ).
+
+### 3. Work with project maintainer for final approval
+
+As denoted in [`governance.md`](https://github.com/openxla/stablehlo/blob/main/docs/governance.md),
+while we work towards instating StableHLO module maintainers, the interim review
+process requires approval from Google project maintainers. A member of the
+StableHLO team will help drive final approval.
+
+### 4. Send PRs for the opset change
+
+Once an RFC is approved, PRs which implement the approved proposal may be sent,
```

Comment:
Consider adding a link to the collective_broadcast implementation PR to continue the examples: https://github.com/openxla/stablehlo/pull/1856